content/general/concepts/what-is-cloud-computing/what-is-cloud-computing.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'What is Cloud Computing?'
+Description: 'Cloud Computing is a term used for on-demand availability of computer system resources, particularly data storage (cloud storage) and computational power without direct active management by the user. Users have to pay only for the services they use. Instead of buying, owning, and maintaining physical data centers and servers, users can access technology services, such as computing power, storage, and databases, on an as-needed basis from a cloud provider. There are many cloud providers in the market providing different services with different pricing models.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+  - 'Cloud'
+  - 'DevOps'
+CatalogContent: # If listing multiple URLs, please put the most relevant one first
+  - 'paths/computer-science'
+---
+
+**Cloud Computing** is a term used for on-demand availability of computer system resources, particularly data storage (cloud storage) and computational power without direct active management by the user. Users have to pay only for the services they use. Instead of buying, owning, and maintaining physical data centers and servers, users can access technology services such as computing power, storage, and databases on an as-needed basis from a cloud provider. There are many cloud providers in the market providing different services with different pricing models.
+
+## History of Cloud Computing
+
+**DARPA (Defence Advanced Research Projects Agency)** awarded MIT $2 million for Project MAC in 1963. The funding required MIT to develop technology that would allow a “computer to be used by two or more people at the same time.” In this case, one of those massive, archaic computers that used reels of magnetic tape for memory and were the precursor to what is now collectively known as the **Cloud Computing**.
+
+## How does Cloud Computing work?
+
+It is divided into two sections, i.e., frontend and backend. They are connected through the Internet. The front end consists of the client’s computer or computer network. Also, the application is essential to access the cloud system. Not all cloud computing systems need to have the same user interface. On the back end of the cloud technology system, various computers, servers, and data storage systems make up the cloud. A cloud computing system could include any computer program, from data processing to video games. Generally, each application will have its dedicated server.
```

Comment:
> It is divided into two sections, i.e., frontend and backend. They are connected through the Internet. The front end consists of the client’s computer or computer network.

Let's use consistent terms and stick with "frontend".

> On the back end of the cloud technology system, various computers, servers, and data storage systems make up the cloud.

Same thing, here. Let's use "backend". Also, the latter part of the sentence seems like a great opportunity to use an unordered list leading with "On the backend, the cloud is made of the following: ...".